### DISCUSSION TITLE
Question ? About Ring

### DISCUSSION THREAD

**Discussion by gdg316** [tsightler/ring-mqtt#434] [Support Q&A] [answered]
Is there anyway to add Ring Components directly to a Zigbee/ZWave Stick ??
Like the Key Pad and Basestation , motion and door/window sensors ??
Or does that stuff need the Ring Cloud ?? Which I am running all through The Ring-MQTT w/video streaming 
I am just curious because it's all ZWave Plus 
Even my first Alert Smoke/CO detectors

**Comment by tsightler** [comment, marked as answer]
Of course it's possible, but then they will no longer be paired to Ring alarm and I'm not sure why you would purchase Ring sensors to just pair them directly as there are cheaper generic Z-wave sensors.  Indeed the First Alert sensors are just compatible with Ring, but you can use them with any Z-wave environment.  Information about pairing the Ring keypad directly with Home Assistant is available, for example:

https://github.com/ImSorryButWho/HomeAssistantNotes/blob/6f05b3f90a979f7118e7c688ce2d01b878227328/RingKeypadV2.md

There's plenty of information available online around these devices and how you can use them with generic Z-wave hubs/sticks.   This isn't really the appropriate forum for those discussions however, so I'd request that you ask those questions on the HA forums or other more appropriate places.  The discussions here are intended for discussions of ring-mqtt only.  Thanks!
